packages/grid/_modules_/grid/components/cell/GridCell.tsx

```diff
@@ -177,6 +180,30 @@ function GridCell(props: GridCellProps) {
     }
   });
 
+  let handleFocus: any = null;
+
+  if (process.env.NODE_ENV === 'test') {
```

Comment:
We should probably also call `other.onFocus` since it can be overridden with `componentsProps`